webviz_ert/__main__.py

```diff
@@ -0,0 +1,114 @@
+import shutil
+import signal
+import sys
+import os
+import logging
+import tempfile
+import yaml
+import pathlib
+import argparse
+from typing import Any, Optional, Dict
+
+from webviz_ert.assets import WEBVIZ_CONFIG
+
+
+logger = logging.getLogger()
+
+
+def handle_exit(
+    *args: Any,
+) -> None:  # pylint: disable=unused-argument
+    # pylint: disable=logging-not-lazy
+    logger.info("\n" + "=" * 32)
+    logger.info("Session terminated by the user.\nThank you for using webviz-ert!")
+    logger.info("=" * 32)
+    sys.tracebacklimit = 0
+    sys.stdout = open(os.devnull, "w")
+    sys.exit()
+
+
+def create_config(
+    project_identifier: Optional[str],
+    config_file: pathlib.Path,
+    temp_config: Any,
+    experimental_mode: bool,
+) -> None:
+    with open(config_file, "r") as f:
+        config_dict = yaml.safe_load(f)
+        for page in config_dict["pages"]:
+            for element in page["content"]:
+                for key in element:
+                    if element[key] is None:
+                        element[key] = {"project_identifier": project_identifier}
+                    else:
+                        element[key]["project_identifier"] = project_identifier
+
+    new_config_dict = config_dict
+
+    def filter_experimental_pages(
+        page: Dict[str, Any], experimental_mode: bool
+    ) -> bool:
+        if "experimental" in page:
+            if page["experimental"]:
+                return experimental_mode
+        return True
+
+    new_config_dict["pages"] = [
+        page
+        for page in new_config_dict["pages"]
+        if filter_experimental_pages(page, experimental_mode)
+    ]
+
```

Comment:
Clean Code -> Vertical order -> function below a function that does the calling. Is here an exception?

Reply:
No, I simply move this function from ert to webviz_ert. I'll clean it up :)

Reply:
Fixed now.